src/ServiceControl.Audit.Persistence.RavenDB/RavenExternalPersistenceLifecycle.cs

```diff
@@ -35,10 +38,21 @@ public async Task Initialize(CancellationToken cancellationToken = default)
             {
                 await initializeSemaphore.WaitAsync(cancellationToken);
 
+                // Look for raven-client-certificate.pfx in same directory as application code
+                var applicationDirectory = Path.GetDirectoryName(Assembly.GetEntryAssembly()!.Location) ?? string.Empty;
```

Comment:
@bording with the change to null coalesce to string, should the `!.Location` be changed to `?.Location`? Docs say it can return null when called from unmanaged code…which I don't see ever happening? But maybe `?.` would be safer?